backend_addon/{{ cookiecutter.__folder_name }}/README.md

```diff
@@ -17,8 +17,22 @@ And to create the Plone site:
 
 ```shell
 make create_site
+
 ```
 
+## Add features using plonecli/bobtemplates.plone
+
+This package provides plonecli/bobtemplates.plone compatible entrypoints to add all kind of subtemplates provided by them.
+
+To use them, you need to use bobtemplates.plone version > X.X.X and run your command as follows:
```

Comment:
```suggestion
To use them, you need to use `bobtemplates.plone` version X.X.X or later, and run the following command.
```

Reply:
Here we will need to set the version number of the new bobtemplate.plone package :)